cmd/build_test.go

```diff
@@ -14,7 +15,32 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const buildUsage = `
+func BuildUsage() string {
```

Comment:
Is this simpler?

```
func BuildUsage() string {
	defaultSocketPath := ""
	if len(launch.DefaultSocketPath()) != 0 {
		defaultSocketPath = fmt.Sprintf(" (default \"%s\")", launch.DefaultSocketPath())
	}

	return fmt.Sprintf(`
Usage:
  build [job name] [flags]

Flags:
      --artifacts-dir string   Path to the host side directory which is mounted into $SD_ARTIFACTS_DIR. (default "sd-artifacts")
  -e, --env stringToString     Set key and value relationship which is set as environment variables of Build Container. (<key>=<value>) (default [])
      --env-file string        Path to config file of environment variables. '.env' format file can be used.
  -h, --help                   help for build
  -i, --interactive            Attach the build container in interactive mode.
  -m, --memory string          Memory limit for build container, which take a positive integer, followed by a suffix of b, k, m, g.
      --meta string            Metadata to pass into the build environment, which is represented with JSON format
      --meta-file string       Path to the meta file. meta file is represented with JSON format.
      --privileged             Use privileged mode for container runtime.
  -S, --socket string          Path to the socket. It will used in build container.%s
      --src-url string         Specify the source url to build.
                               ex) git@github.com:<org>/<repo>.git[#<branch>]
                                   https://github.com/<org>/<repo>.git[#<branch>]
      --sudo                   Use sudo command for container runtime.
      --vol strings            Volumes to mount into build container.

`, defaultSocketPath)
}
```